File: ContasOnlineModel/Modelo/Despesa.cs
using System;
using System.Collections.Generic;
using System.Text;
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace ContasOnlineModel.Modelo
{
    public class Despesa
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public String _id { get; set; }
        public Boolean UtilizouCartaoDeCredito { get; set; }
        public Cartao CartaoUtilizado { get; set; }
        public Boolean DespesaParcelada { get; set; }
        public Int32 QuantidadeDeParcelas { get; set; }
        public Conta ContaDeSaida { get; set; }
        public Categoria Categoria { get; set; }
        public Decimal Valor { get; set; }
        public String Observacao { get; set; }
        public DateTime DataDeCadastro { get; set; }

    }
}

File: ContasOnlineApp/src/model/despesa.ts
import { DateTime } from 'ionic-angular';

export class Model {

    constructor(objeto?) {
        Object.assign(this, objeto);
    }
  
  }
  
   //Classe de Despesa, esta deve ser utilizada na pagina de resitro de despesa
  export class Despesa extends Model {

    _id: string;
    UtilizouCartaoDeCredito: boolean;
    CartaoUtilizado: object;
    DespesaParcelada: boolean;
    QuantidadeDeParcelas: number;
    ContaDeSaida: object;
    Categoria: object;
    Valor: number;
    Observacao: string;
    DataDeCadastro: DateTime;

  }